pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/internal/PrettyPrintingUtil.java

```diff
@@ -77,4 +78,12 @@ public static String kindName(ASTAnyTypeDeclaration decl) {
         return "class";
     }
 
+    public static String prettyImport(ASTImportDeclaration importDecl) {
```

Comment:
Just a general thought (does not need to be part of this PR, this class is internal anyway): Wouldn't those pretty-print methods fit into the nodes themselves? like `importDecl.prettyPrint()`. Or do you think, this opens the door for adding all kind of different methods into the AST nodes, which could make them eventually fat?

Reply:
I think for now it's more prudent to make those methods external to the node, because they're mostly relevant in rules and error messages anyway. Otherwise we'd probably have to make stronger stability guarantees on the output of these methods...